src/main/java/org/jabref/gui/actions/ActionHelper.java

```diff
@@ -30,10 +32,15 @@ public static BooleanExpression isFieldSetForSelectedEntry(Field field, StateMan
     }
 
     public static BooleanExpression isAnyFieldSetForSelectedEntry(List<Field> fields, StateManager stateManager) {
-        BibEntry entry = stateManager.getSelectedEntries().get(0);
-        return Bindings.createBooleanBinding(
-                () -> entry.getFields().stream().anyMatch(fields::contains),
-                entry.getFieldsObservable(),
-                stateManager.getSelectedEntries());
+        ObservableList<BibEntry> selectedEntries = stateManager.getSelectedEntries();
+
+        ObjectBinding<BibEntry> entry = Bindings.valueAt(selectedEntries, 0);
+        return Bindings.createBooleanBinding(() -> {
+            if (entry.get() == null) {
+                return false;
+            } else {
+                return entry.get().getFields().stream().anyMatch(fields::contains);
+            }
+        }, selectedEntries);
```

Comment:
I'm missing `entry.getFieldsObserable()` here. Is this intended?

Reply:
Good catch. This is a problem - it might not be available (that's why I need the complex code above). I'm not sure what side effects this might have, I need you guys' expertise here.

Reply:
Not sure but maybe the following works:
```
EasyBind.monadic(entry)
             .flatMap(BibEntry:getFieldsObservable)
             .map(entryFields -> entryFields.anyMatch(...))
```

Reply:
When does this situation occur? I.e. the fields of the entry change dynamically?

Reply:
This binding should update in the case when the selected entry changes as well as when the fields of the currently selected entry change